### PR TITLE
Add URL filter example

### PR DESCRIPTION
**Pull request by jadb**
This was proposed by @lorenzo.

Suggestions on improving the description are welcome :)

### CODE REVIEW

File: en/development/routing.rst [hunk 1]
```diff
@@ -1112,6 +1112,21 @@ URL filters allow you to implement features like persistent parameters::
 
 Filter functions are applied in the order they are connected.
 
+Another use case is changing a certain route on runtime (plugin routes for example):
```
Comment (markstory) [inline en/development/routing.rst]: Should end with `::` to make a code block.

**Comment by cake17** [comment]
french translation has been added

File: en/development/routing.rst [hunk 2]
```diff
@@ -1112,6 +1112,22 @@ URL filters allow you to implement features like persistent parameters::
 
 Filter functions are applied in the order they are connected.
 
+Another use case is changing a certain route on runtime (plugin routes for
+example)::
+
+    Router::addUrlFilter(function ($params, $request) {
+        if (empty($params['plugin']) || $params['plugin'] !=== 'MyPlugin') || empty($params['controller'])) {
```
Comment (saeideng) [inline en/development/routing.rst]: missing a `(`
`!===` or `!==` ?

Comment (dereuromark) [inline en/development/routing.rst]: The first doesnt exist :)

Comment (jadb) [inline en/development/routing.rst]: Thanks! Fixed :)

Comment (saeideng) [inline en/development/routing.rst]: if (empty($params['plugin']) || $params['plugin'] !== 'MyPlugin' `)` || empty($params['controller'])) {

**Pull request merged**